media/css/externalpages/pocket/components/_add.scss

```diff
@@ -0,0 +1,165 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '../utils/variables';
+
+$image-path: '/media/img/externalpages/pocket';
+
+.add-page {
+    .add-header {
+        background-color: #f0f0f0;
+    }
+
+    h2 {
+        margin-bottom: $spacing100;
+        font-size: $size200;
+        color: $color-text-dark-grey;
```

Comment:
For this section and others, It would be a good idea to follow the alphabetical ordering that we use in bedrock